e2e/actors/base.js

```diff
@@ -184,7 +184,7 @@ module.exports = {
         await this.goToPage(`${baseUrl}/cases/case-details/${caseId}`);
       }, signedInSelector);
     }
-    await this.waitForSelector('.ccd-dropdown');
+    await this.waitForSelector('#next-step');
```

Comment:
Not sure where this wait for selector method is defined on my phone but can we instead use wait for element and remove the async nature?

Reply:
`waitForSelector()` looks to be a FPLA defined method, but reading the code it looks like it's just doing the equivalent of a `waitForElement()` anyway, just without throwing the error (possibly to allow the `retryUntilExists()` code upstream to handle it instead of Codecept)
```
  async waitForSelector(locator, sec) {
    const helper = this.getHelper();

    try {
      if (this.isPuppeteer()) {
        const waitTimeout = sec ? sec * 1000 : helper.options.waitForTimeout;
        const context = await helper._getContext();
        return await context.waitForSelector(locator, {timeout: waitTimeout});
      } else {
        const waitTimeout = sec ? sec : helper.options.waitForTimeout;
        return await helper.waitForElement(locator, waitTimeout);
      }
    } catch (error) {
      return undefined;
    }
  }
```
So looks like it should be fine to change - I'll do that now

Reply:
done